Firmware/cmdqueue.cpp

```diff
@@ -251,6 +252,19 @@ void cmdqueue_dump_to_serial()
 static const char bufferFull[] PROGMEM = "\" failed: Buffer full!";
 static const char enqueingFront[] PROGMEM = "Enqueing to the front: \"";
 
+
+void enquecommandf(const char *fmt, ...)
```

Comment:
proposal for another name: enqueuecommandf_P() since the fmt is from PROGMEM. This is how I've tried naming anything PGM related in the past.

Reply:
Done ✔️

Reply:
Note sure about changing "enque" to "enqueue" without also applying that to the other "enque" functions.

Reply:
Oh, I didn't see that typo. Good catch. I think we should name it like the other functions and just deal with renaming everything at a later time (if ever).